### PR TITLE
Merge FAQ with main-menu banner support section

### PR DESCRIPTION
**Pull request by mangomizer**
Crucial information on eligibility was in FAQ, which could be missed.

### CODE REVIEW

**Comment by mangomizer** [comment]
Should clarify that I think certain sections on this article should be shuffled around because it doesn't seem very logical (why is the "how to requesting support" at the bottom of the page?

File: wiki/Tournaments/Official_support/en.md [hunk 1]
```diff
@@ -93,7 +100,7 @@ In rare circumstances, some long-running community tournaments may qualify for i
 - The banner must clearly display the logo, motif AND name of your tournament, plus any information about the ongoing stage or section that is being advertised.
 - The banner must be cleanly designed with an emphasis on being mostly transparent and relatively unobtrusive visually.
 
-Please do not submit a main menu banner with your original request—you will be asked and pointed to this section if you are authorized to make use of one.
+ Please do not submit a main menu banner with your original request — you will be asked and pointed to this section if you are authorized to make use of one.
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:103]: Articles generally don't put spaces around em dashes. News posts have somewhat relaxed rules on writing style, which is why they do have spaces there by consensus, but no such thing has been decided for regular articles.

```suggestion
Please do not submit a main menu banner with your original request—you will be asked and pointed to this section if you are authorized to make use of one.
```

File: wiki/Tournaments/Official_support/en.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ The [osu! team](/wiki/People/The_Team) runs a program where they may opt to prov
 Officially supported tournaments have access to:
 
 - **Profile badge prizes for first-place winners**
-- Assessment for a main-menu banner during major matches (generally quarterfinals and above), subject to availability and criteria (see the [FAQ](#frequently-asked-questions) section below)
+- Assessment for a main-menu banner during major matches (generally quarterfinals and above), subject to availability and criteria (see the [relevant section](#Requesting-main-menu-banner-support) below)
```
Comment (Walavouchey) [inline wiki/Tournaments/Official_support/en.md:10]:
```suggestion
- Assessment for a main-menu banner during major matches (generally quarterfinals and above), subject to availability and criteria (see the [relevant section](#requesting-main-menu-banner-support) below)
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by cl8n — APPROVED** [review]
looks good to me, make sure nothing important links to `#main-menu-advertise` though (not sure if that was originally added on purpose for a link somewhere)

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**